Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.h

```diff
@@ -388,6 +388,8 @@ namespace AzToolsFramework
             // A container for mapping Templates' file paths to their Template ids.
             AZStd::unordered_map<AZ::IO::Path, TemplateId> m_templateFilePathToIdMap;
 
+            AZStd::unordered_map<AZ::EntityId, AZ::IO::Path> m_entityIdToHashedPathMap;
```

Comment:
What is the meaning of path here? Path to what?

Reply:
I'll investigate! Looping in @AMZN-koppersr @srikappa-amzn @michabr as this is their domain.

Reply:
During the loading of prefab instances(that includes every entity inside any prefab including levels), we hash the path of the entity as seen from the top-most prefab (root prefab) and use it to generate entity id. So, consistent paths yield consistent ids. This logic can be found [here](https://github.com/o3de/o3de/blob/38261d08007a1ad14135cf790cb855ccca4d8595/Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceEntityIdMapper.cpp#L46-L50) . For the network prefab use case, entity ids needed to be shared by 2 prefabs to keep entity references intact. So, instead of fetching paths from a unique owning instance, they will be fetched from this map if the relationship is one-to-many.

A comment here could say :

"A map of entity id to hashed path used for generation of entity id during deserialization. This map is needed when there is a many-to-one relationship between entity ids and hashed paths."